builder/dockerfile/builder.go

```diff
@@ -22,15 +22,16 @@ import (
 )
 
 var validCommitCommands = map[string]bool{
```

Comment:
The PR does not change this code. If you want to clean up nearby code, I it think should be done in a separate PR.